include/swift/Option/FrontendOptions.td

```diff
@@ -174,9 +174,6 @@ def disable_objc_attr_requires_foundation_module :
   HelpText<"Disable requiring uses of @objc to require importing the "
            "Foundation module">;
 
-def enable_resilience : Flag<["-"], "enable-resilience">,
-   HelpText<"Compile the module to export resilient interfaces for all "
-            "public declarations by default">;
```

Comment:
We need to keep this as an alias, at least for a little while, since people are already using it outside this repo.